### ISSUE TITLE
GameGuru MAX - Request - Game Menu Advanced Graphics Options On/Off

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [Dark-Basic-Software-Limited/GameGuruRepo#2515]
Now that we have access to more advanced graphics features via the WickedEngine in MAX, it would be great if we could give people, who may play the games we create, the option to turn on or off advanced graphics features individually depending on their PCs. They should be able to turn on or off things like:

- FXAA
- SSR
- Screen Space Reflections (if the developer had turned these on)
- Tesselation
- Adjust MSAA
- Even adjust Gamma to their liking

This may become even more important as TGC considers adding even more WickedEngine graphic features to MAX.

**Comment by DarthPickles666** [comment]
Yeah, I think this definitely needs to happen. I understand that it's for simplicity, but it would definitely make it better.

**Comment by SampeyFS** [comment]
Yes

**Comment by SampeyFS** [comment]
Yes exactly these need to be placed

**Comment by Kitakazi** [comment]
pls god

**Comment by thatmodelguy** [comment]
Having the option to change the resolution in the settings would be good too.

**Comment by LightofAlfheim** [comment]
need this!
